avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastDeserializerGeneratorBase.java

```diff
@@ -0,0 +1,303 @@
+package com.linkedin.avro.fastserde;
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.Lists;
+import com.google.common.hash.HashFunction;
+import com.google.common.hash.Hashing;
+import com.sun.codemodel.JBlock;
+import com.sun.codemodel.JCodeModel;
+import com.sun.codemodel.JDefinedClass;
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.ListIterator;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ThreadLocalRandom;
+import javax.tools.JavaCompiler;
+import javax.tools.ToolProvider;
+import org.apache.avro.Schema;
+import org.apache.avro.io.parsing.Symbol;
+import org.apache.log4j.Logger;
+
+
+public abstract class FastDeserializerGeneratorBase<T> {
+  public static final String GENERATED_PACKAGE_NAME = "com.linkedin.avro.fastserde.deserialization.generated";
+  public static final String GENERATED_SOURCES_PATH = "/com/linkedin/avro/fastserde/deserialization/generated/";
```

Comment:
File.pathSeparator? (i realize this code is far from being windows-compatible, but still)

Reply:
You meant `File.seperator`, right?